documentation/staking-governance.md

```diff
@@ -29,21 +29,37 @@ The voting power for a locked amount of tokens is computed as follows:
 - Generally, locked tokens provide a voting power proportional to the amount of locked tokens. Concretely, 1 locked LSK provides one unit of voting power.
 - In case a user pauses their locking period countdown, they receives a boost of the voting power meaning the voting power is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)`. Hence, the voting power can be increased by up to 200%.
 
```

Comment:
Comment about lines 12-23: Since there is a lot of content in bullet points, perhaps would be nice to use bold letters in some places such as "modify locking position" or "fast unlock" so that it's easier to parse the content.